src/components/Heading/Heading.stories.tsx

```diff
@@ -67,9 +67,14 @@ export const WithControls: Story = {
       <Heading
         {...args}
         bgColor="brand.primary"
-        color="ui.typography.inverse.heading"
+        color="ui.white"
         px="s"
         py="xs"
+        __css={{
+          h2: {
+            color: "ui.white",
+          },
+        }}
```

Comment:
Why is this necessary?

Reply:
Hmm, I am not sure. I recall that the `color` value was not being passed properly to the `<h>` element, but that seems to be working fine now. I'll get rid of this.